stake/src/epoch.rs

```diff
@@ -2,10 +2,8 @@ use anyhow::{anyhow, Error};
 
 /// Epoch represents a given epoch for Penumbra and is used
 /// for calculation of staking exchange rates.
-#[derive(Debug, Eq, PartialEq)]
-pub struct Epoch {
-    epoch: u64,
-}
+#[derive(Debug, Eq, PartialEq, Clone)]
+pub struct Epoch(pub u64);
```

Comment:
non-blocking: I think it could be helpful to change the Epoch definition to
```rust
pub struct Epoch {
    index: u64,
    duration: u64,
}
```
so that the `Epoch` is self-contained enough to be able to compute data like its start and end heights, and check whether a particular height is contained within the epoch.  My guess is that this would make stuff like https://github.com/penumbra-zone/penumbra/pull/299/files#r771231900 easier to manage.

The only complication is that we might want to serialize into the database by `Epoch::index()`, so deserialization would require having the epoch duration on hand.